src/sonic-config-engine/tests/test_minigraph_case.py

```diff
@@ -97,9 +97,16 @@ def test_minigraph_vlan_members(self):
         self.assertEqual(output.strip(), "{('Vlan1000', 'Ethernet8'): {'tagging_mode': 'untagged'}}")
 
     def test_minigraph_vlan_interfaces(self):
-        argument = '-m "' + self.sample_graph + '" -p "' + self.port_config + '" -v "VLAN_INTERFACE.keys()|list"'
+        argument = '-m "' + self.sample_graph + '" -p "' + self.port_config + '" -v "VLAN_INTERFACE"'
         output = self.run_script(argument)
-        self.assertEqual(output.strip(), "[('Vlan1000', '192.168.0.1/27'), 'Vlan1000']")
+        expected_table = {
```

Comment:
nit: can you add a new test case? Not sure if there are many test cases that would test .keys() method.

Reply:
will do